message-store/message-store.bal

```diff
@@ -0,0 +1,29 @@
+import ballerina/http;
+
+type FirebaseResponse record {|
+    string address?;
+    record {}[] results?;
+|};
+
+type GeoCodeResponse record {|
+    record {}[] results;
+|};
+
+final http:Client geoCodingClient = check new ("http://api.maps.googleapis..com.balmock.io");
+final http:Client firebaseClient = check new ("http://api.mapsproject.firebase.com.balmock.io", 
+    {httpVersion: http:HTTP_1_1});
+
+service /api on new http:Listener(8080) {
+    
+    resource function get location(string address) returns GeoCodeResponse|error {
+        FirebaseResponse location = check firebaseClient->/location/[address]/["location.json"]();
+        if !location.hasKey("address") {
+            GeoCodeResponse geocode = check geoCodingClient->/maps/api/geocode/'json(place=address);
+            json _ = check firebaseClient->/location/[address]/["location.json"].post({
```

Comment:
can we use `var _ = `

Reply:
will `.post(geocode);` work?

Reply:
We cannot use `var` because from the R.H.S the compiler cannot identify the contextually expected type.